core/templates/dev/head/pages/BaseSpec.js

```diff
@@ -20,20 +20,21 @@ describe('Base controller', function() {
   beforeEach(module('oppia'));
 
   describe('BaseCtrl', function() {
-    var scope, ctrl, $httpBackend;
+    var scope, ctrl, rof;
 
-    beforeEach(inject(function($rootScope, $controller) {
+    beforeEach(inject(function($rootScope, $controller, $injector) {
       scope = $rootScope.$new();
       ctrl = $controller('Base', {
         $scope: scope,
         alertsService: null,
         messengerService: null
       });
+      rof = $injector.get('RuleObjectFactory');
     }));
 
     it('should have matching classifier constants',
         inject(function($injector) {
-      expect($injector.get('DEFAULT_CLASSIFIER_RULE').rule_type).toEqual(
+      expect(rof.createNewClassifierRule().type).toEqual(
```

Comment:
Oh, no, this isn't what I meant. I meant delete this file and create a RuleObjectFactorySpec.js in the right place, and have this be one of its tests, especially since this has nothing to do with the base controller.